docs/src/man/getting_started.md

```diff
@@ -236,6 +236,26 @@ julia> tail(df)
 │ 6   │ 999   │ 10    │ 500   │
 ```
 
+Also notice that when `DataFrame` is printed to the console or rendered in HTML (e.g. in Jupyter Notebook) you get an information about type of elements held in its columns. For example in this case:
+
+```jldoctest dataframe
+julia> DataFrame(a = 1:2, b = [1.0, missing],
+                 c = categorical('a':'b'), d = [big(1.0), missing])
+2×4 DataFrame
+│ Row │ a     │ b        │ c            │ d         │
+│     │ Int64 │ Float64⍰ │ Categorical… │ BigFloa…⍰ │
```

Comment:
Actually it is my bug in the implementation as `BigFloat` fits only `Union{BigFloat, Missing}` does not fit, but it could be printed as `BigFloat⍰` - I will make a separate PR.

AFAIK all major types fit correctly. I will change here to `Rational{Int64}` which does not fit and will not fit.